worldwind-gt-geopkg/src/main/java/gov/nasa/worldwind/geopkg/mosaic/GeoPackageReader.java

```diff
@@ -90,21 +90,44 @@
     protected File sourceFile;
 
     protected Map<String, TileEntry> tiles = new HashMap<String, TileEntry>();
+    
+    // TODO: store these gridRanges in the TileEntry class
+    protected Map<String, GridEnvelope2D> gridRanges = new HashMap<String, GridEnvelope2D>();
 
     public GeoPackageReader(Object source, Hints hints) throws IOException {
         coverageFactory = CoverageFactoryFinder.getGridCoverageFactory(this.hints);
```

Comment:
Good eyes! It appears the GeoPackageReader is intentionally using the "defaultHints" provided in the base class. The rationale for this is unknown.